bin/detail/call.py

```diff
@@ -63,15 +63,16 @@ def call(call_args, logging, cache_file='', ignore=False, sleep=0):
   for i in call_args:
     pretty += '  `{}`\n'.format(i)
   pretty += ']\n'
-  print(pretty)
+  if logging.verbosity == 'full':
```

Comment:
Same as below - I picked an option that provides most (but not more) information required for the developer in day-to-day scenario leaving full output for either `--verbose` or `--verbose-level=full` cases.  Of course a can fix it and make the implementation follow the issue literally.